logging/log_config_test.go

```diff
@@ -26,6 +26,7 @@ func createTestRegistry(loggerNames []string) *loggerRegistry {
 }
 
 func TestValidatePattern(t *testing.T) {
+	// logger pattern matching
```

Comment:
[nit] Could you [table-drive](https://go.dev/wiki/TableDrivenTests) this test and parallelize all subtests?